.travis.yml

```diff
@@ -9,5 +9,5 @@ install:
   - composer install
 
 script:
-  - make check-linting
-  - make test
\ No newline at end of file
+  - composer lint
+  - composer test
```

Comment:
They actually work (tried before commiting) and the CI execute them too so I don't think we need the `run-script`.